### ISSUE TITLE
Cambria Math load error: unpack requires a buffer of 2 bytes

### ISSUE THREAD

**Issue by rainyl** [cdelker/ziafont#1]
Firstly, really thanks for your great works!!!
but when loading `Cambria Math`, some errors occurred.
``` python
import ziafont as zf

font = zf.Font("../data/fonts/CambriaMath.ttf")
```

and the output:
```console
File ziafont/font.py:47, in Font.__init__(self, name)
     45 self.tables = {}
     46 self.features = FontFeatures()
---> 47 self.info = self._loadfont()  # Load in all the font metadata
     48 self._glyphs: Dict[int, Union[SimpleGlyph, CompoundGlyph]] = {}
     49 self._glyphids: Dict[str, int] = {}

File ziafont/font.py:62, in Font._loadfont(self)
     60 self.gpos = None
     61 if 'GPOS' in self.tables:
---> 62     self.gpos = gpos.Gpos(self.tables['GPOS'].offset, self.fontfile)
     64 self.gsub = None
     65 if 'GSUB' in self.tables:

File ziafont/gpos.py:59, in Gpos.__init__(self, ofst, fontfile)
     57 self.lookups = []
     58 for i in range(lookupcnt):
---> 59     self.lookups.append(GposLookup(
     60         self.fontfile.readuint16() + lookuplisttableloc,
...
     26 if offset:
     27     self.seek(offset)
---> 28 return struct.unpack('>H', self.read(2))[0]

error: unpack requires a buffer of 2 bytes
```

and finally, I attached the above font I used, maybe it can help you.

[CambriaMath.zip](https://github.com/cdelker/ziafont/files/12639714/CambriaMath.zip)

**Comment by cdelker** [comment]
Thanks for reporting this. I found a mistake when reading multiple [GPOS Extension Tables](https://learn.microsoft.com/en-us/typography/opentype/spec/gpos#lookuptype-9-extension-positioning). I also fixed an issue caused by CambriaMath not having a DFLT language defined. The font is now working in my tests for ziafont and ziamath. 

Should be fixed in 4e0914e2.

**Comment by rainyl** [comment]
Great! it worked. 😄
